src/en/administration/troubleshooting.md

```diff
@@ -0,0 +1,60 @@
+# Troubleshooting
+
+Different problems that can occur on a new instance, and how to solve them.
+
+## General
+
+### Various problems
+
+Use `docker-compose logs -f lemmy` in your installation folder on the server to view the logs. You can also do `docker-compose logs -f lemmy lemmy-ui pictrs` to get logs from different services.
+
+If that doesn't give enough info, try changing the line `RUST_LOG=error` in `docker-compose.yml` to `RUST_LOG=info` or `RUST_LOG=verbose`, then do `docker-compose restart lemmy`.
+
+### Creating admin user doesn't work
+
+Make sure that websocket is working correctly, by checking the browser console for errors. In nginx, the following headers are important for this:
+
+```
+proxy_http_version 1.1;
+proxy_set_header Upgrade $http_upgrade;
+proxy_set_header Connection "upgrade";
+```
+
+### Rate limit error when many users access the site
+
+Check that the headers `X-Real-IP` and `X-Forwarded-For` are sent to Lemmy by the reverse proxy. Otherwise, it will count all actions towards the rate limit of the reverse proxy's IP. In nginx it should look like this:
+
+```
+proxy_set_header X-Real-IP $remote_addr;
+proxy_set_header Host $host;
+proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+```
+
+## Federation
+
+### Other instances can't fetch local objects (community, post, etc)
+
+Your reverse proxy (eg nginx) needs to forward requests with header `Accept: application/activity+json` to the backend. This is handled by the following lines:
```

Comment:
Also link to the [nginx template](https://github.com/LemmyNet/lemmy/blob/main/ansible/templates/nginx.conf) somewhere here.

Reply:
Okay, added.